sdk/openai/openai-assistants/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../../tsconfig.package",
+  "compilerOptions": {
+    "outDir": "./dist-esm",
+    "declarationDir": "./types",
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
+    "lib": ["esnext", "dom"],
+    "rootDir": "."
+  },
+  "ts-node": { "esm": true },
+  "include": ["src/**/*.ts", "./test/**/*.ts"]
+}
```

Comment:
I think there is other configurations missing here, especially to enable writing samples, feel free to copy the one in openai.